spec/models/friendship_spec.rb

```diff
@@ -0,0 +1,40 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe Friendship, type: :model do
+  let(:friendship) { build(:friendship) }
+
+  it 'has status equal to 0 by default' do
+    expect(friendship.status).to be(0)
+  end
+
+  it 'allows status to be changed with valid entry' do
```

Comment:
In updating the `friendship` status from `0` to `1` later on (when a friend request is accepted), it's not a good practice to simply:

```ruby
friendship.status = 1
```
The more proper way to do this later on is:

```ruby
friendship.update(status: 1)
```

I suppose we are going to need to have an `update` action in the `FriendshipsController` which is going to take care of this accepting a friend request for us. Please see my annotation on the `friendship.rb` model for additional suggestions.